content/maxwell1_fundamentals/maxwells_equations/maxwells_equations_in_homogeneous_media/frequency_domain/dipole_sources/electric_dipole/analytic_solution.rst

```diff
@@ -8,3 +8,81 @@ Analytic Solution
     Purpose here
 
```

Comment:
could you add a purpose statement here? Also, there are lots of equations, any that should be in the equation bank?